shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -120,40 +126,68 @@ public int run() throws IOException {
         + String.format(mInfoFormat, "Miss", cacheMissPercentage));
 
     mPrintStream.println("\nLogical Operations: ");
-    printMetric(MasterMetrics.DIRECTORIES_CREATED, "Directories Created", false);
-    printMetric(MasterMetrics.FILE_BLOCK_INFOS_GOT, "File Block Infos Got", false);
-    printMetric(MasterMetrics.FILE_INFOS_GOT, "File Infos Got", false);
-    printMetric(MasterMetrics.FILES_COMPLETED, "Files Completed", false);
-    printMetric(MasterMetrics.FILES_CREATED, "Files Created", false);
-    printMetric(MasterMetrics.FILES_FREED, "Files Freed", false);
-    printMetric(MasterMetrics.FILES_PERSISTED, "Files Persisted", false);
-    printMetric(MasterMetrics.NEW_BLOCKS_GOT, "New Blocks Got", false);
-    printMetric(MasterMetrics.PATHS_DELETED, "Paths Deleted", false);
-    printMetric(MasterMetrics.PATHS_MOUNTED, "Paths Mounted", false);
-    printMetric(MasterMetrics.PATHS_RENAMED, "Paths Renamed", false);
-    printMetric(MasterMetrics.PATHS_UNMOUNTED, "Paths Unmounted", false);
+    printMetric(MetricsSystem.getMasterMetricName(MasterMetrics.DIRECTORIES_CREATED),
```

Comment:
There is no easy way to iterate over these? Could we just output the values in the map, that would also avoid the need to expose `getMasterMetricName`?

Reply:
Previously we want `fsadmin report metrics` similar to the master web ui metrics page, that's why we add those complicate logic to add `nick name` to those metrics. I agreed that we can modify to directly output contents in the map and maybe sort the metrics according to prefix (master/cluster/worker)